src/js/edu-benefits/utils/helpers.js

```diff
@@ -14,6 +15,15 @@ export const chapterNames = {
   review: 'Review'
 };
 
+export const benefitsLabels = {
+  chapter33: <p>Post-9/11 GI Bill (Chapter 33)<br/><a href="/education/gi-bill/post-9-11/" target="_blank">Learn more</a></p>,
```

Comment:
Turns out I just need to restart `npm run watch` after files are renamed...which is odd, but that showed me the proper files that specifically referred to `helpers.js` (with the extension).